core/apc-emulation.php

```diff
@@ -183,6 +201,16 @@ static public function DeleteEntry($sCache)
 		return true;
 	}
 
+	/**
+	 * Check if cache key exists
+	 * @param $sKey
+	 * @return bool
+	 */
+	static public function ExistsOneFile($sKey) {
+		if (is_file(self::GetCacheFileName('-'.$sKey))) return true;
+		else return is_file(self::GetCacheFileName($sKey));
```

Comment:
Could also be rewritten as this if preferred:
```suggestion
		return is_file(apcFile::GetCacheFileName('-' . $sKey)) || is_file(apcFile::GetCacheFileName($sKey));
```

Reply:
Yes please